src/proxy.ts

```diff
@@ -70,6 +75,28 @@ export function createProxyEndpoints(app: Application, options: Options) {
     await fetchAndRespond('GET', req.path, req.query, req.headers, req.body, res, options)
   })
   app.post(POST_PROXY_ENDPOINTS, async (req, res) => {
+    if (options.filteringActive && req.path.startsWith('/bytes') && req.method === 'POST') {
+      const bodyBuffer = Buffer.from(req.body)
+      const userMessage = JSON.parse(bodyBuffer.toString('utf8')) as UserMessage
```

Comment:
trycatch would be nice for parsing

Reply:
Fix: https://github.com/Solar-Punk-Ltd/gateway-proxy/blob/lb-ai/src/proxy.ts#L80-L92